processors/base/helpers/__init__.py

```diff
@@ -4,20 +4,28 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
 import re
 import json
 import uuid
 import string
 import logging
 import datetime
 import urlparse
+import csv
+import iso3166
+
+from fuzzywuzzy import process, fuzz
```

Comment:
They don't automatically import the `fuzz` or `process` methods, so you'd need to do:

```python
import fuzzywuzzy.fuzz
import fuzzywuzzy.process
```

Also, remember of moving it to the import block close to `import csv`, `import iso3166`, etc.